ws-nextjs-app/cypress/e2e/languagesPage/index.cy.ts

```diff
@@ -33,7 +34,11 @@ const staticPageTestSuites = [
 ];
 
 runTestsForPage({
-  testSuites: [...homePageTestSuites, ...staticPageTestSuites],
+  testSuites: [
+    ...localEnvironmentTestSuites,
+    ...testEnvironmentTestSuites,
+    ...liveEnvironmentTestSuites,
```

Comment:
Whilst this is fine, we didn't really need to do this. We could declare all tests in an array - `testSuites`, and pass it into the runner.